### PR TITLE
Add devtest skill

### PR DESCRIPTION
**Pull request by kayjoosten**

### CODE REVIEW

File: plugins/aimate/skills/dev-test/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
+---
+name: devtest
+description: Use when the user wants to know how to manually test code changes on a branch - analyzes the diff, identifies what changed, and produces a step-by-step guide a real person can follow to verify the feature or fix works, including any required setup like fixtures, mock SPs, or config. Also use when asked for a test plan, QA checklist, or "how do I test this PR".
+metadata:
+  author: Kay Joosten <kay.joosten@dawn.tech>
+  version: 1.1.0
+agents:
```
Comment (mroest) [inline plugins/aimate/skills/dev-test/SKILL.md]: Waarom een agents opnemen? Devs op OpenCode worden nu beperkt? Daarbij als we het enkel van Dawn bekijken supported we alleen copilot/vsc? Het weglaten van de agents is dan vriendelijker

File: plugins/aimate/skills/dev-test/SKILL.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+   - URL contains `gitlab.com` or a self-hosted GitLab domain → `provider = "gitlab"`
+   - If ambiguous, ask the user
+2. Verify GitLab MCP tools are available in the current session.
+   If missing → **do not proceed silently**. Use one of the following approaches:
+
+   **Option A — inline guidance** (when the user just needs quick instructions):
+   > GitLab MCP is not installed. See the `install-mcp-server` skill for setup instructions, then re-run this skill with the MR URL.
+
+   **Option B — dispatch as subagent** (when running inside an automated workflow):
+   > Dispatch the `install-mcp-server` skill as a subagent with input: `provider = "gitlab"`. Wait for confirmation that installation succeeded, then continue from step 3 below.
```
Comment (mroest) [inline plugins/aimate/skills/dev-test/SKILL.md]: undefined skill ref `install-mcp-server`

File: plugins/aimate/skills/test-pr-guide/SKILL.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+- **Are there non-user-visible changes (logging, metrics, internal refactors)?** If so, note this explicitly — the test guide must describe how to verify them indirectly.
+
+### Step 2 — Identify test scenarios
+
+For each behavior changed:
+- **Happy path** — the thing that should now work
+- **Edge cases** — boundary conditions touched by the change
+- **Regression** — existing behavior that should still work
+
+### Step 3 — Discover project context
```
Comment (mroest) [inline plugins/aimate/skills/test-pr-guide/SKILL.md:70]: Intensieve stap die (denk ik) veel eenvoudiger geschreven kan worden in 1 of 2 zinnen. Agents zijn slim genoeg om uit te vinden hoe een project werkt. Files als AGENTS.md worden sowieso geladen en hoef je niet apart te noemen.

File: plugins/aimate/skills/test-pr-guide/SKILL.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+| Mock services | External services referenced in `.env.example` that need a local stub |
+| Config / parameters | Feature flags, env vars that must be set to enable the changed behavior |
+| Database state | Specific records that must exist before the scenario can run |
+| Frontend build | Asset compilation step required before testing UI changes |
+| External service | Third-party integrations referenced in the diff |
+| Security / authz | Required roles, permissions, or auth state for the scenario to be reachable |
+
+### Step 5 — Write the guide
+
+Structure the output as follows:
```
Comment (mroest) [inline plugins/aimate/skills/test-pr-guide/SKILL.md:90]: Heb je de template getest? Ik kan mij voorstellen dat de agent er regelmatig naast zit en minder goed kan bepalen wat als template gebruikt moet worden en wat niet. BV dat je ineens het kopje ## Tips for Good Guides in je resultaat krijgt.

**Review by mroest — APPROVED** [review]
👍
